bootstrapvz/base/manifest-schema.yml

```diff
@@ -47,10 +48,10 @@ properties:
         enum: [i386]
       bootloader:
         enum:
-        - pvgrub
-        - grub
-        - extlinux
-        - none
+          - pvgrub
+          - grub
+          - extlinux
+          - none
```

Comment:
👍  for this change, both versions are valid, but I really don't like the unindented version either, it's ambiguous.